cpp/src/stream_compaction/distinct_count.cu

```diff
@@ -150,15 +150,23 @@ cudf::size_type distinct_count(table_view const& keys,
       stream.value()};
 
     auto const iter = thrust::counting_iterator<cudf::size_type>(0);
-    // when nulls are equal, insert non-null rows only to improve efficiency
+    // when nulls are equal, only insert those rows that are not all null to improve efficiency
```

Comment:
```suggestion
    // when nulls are equal, only insert those rows that are not all valid to improve efficiency
```
Did you mean `not all valid` here? An null bitmask means all rows are valid and not all null. The wording is a bit confusing here.

Reply:
We have a table here, so "not all valid" here means all elements at the same row in all columns are valid.

Reply:
Updated the comment to clarify.